pkg/query/selection.go

```diff
@@ -36,6 +36,8 @@ const (
 	Separator string = "and"
 )
 
+type TemplateParameters map[string]interface{}
```

Comment:
remove this, use map in the one place it is used

Reply:
Done